pkgs/data/fonts/kanit/default.nix

```diff
@@ -12,12 +12,16 @@ stdenv.mkDerivation rec {
   };
 
   installPhase = ''
+    runHook preInstall
+
     mkdir -p $out/share/doc/${pname}/css/ $out/share/fonts/{opentype,truetype}
 
     cp $src/OFL.txt $src/documentation/{BRIEF.md,features.html} $out/share/doc/${pname}
     cp $src/documentation/css/fonts.css $out/share/doc/${pname}/css
     cp $src/fonts/otf/*.otf $out/share/fonts/opentype
     cp $src/fonts/ttf/*.ttf $out/share/fonts/truetype
+
+    runHook postInstall
```

Comment:
seems unrelated, maybe this should be in a separate commit or a separate pull request?

Reply:
Ouch. Edited the wrong file. I'm maintaining this file too so I was using it as reference. Since this is the case, I do think I'd prefer they stay similar; I will do as a separate commit.